src/agoradatatools/gx.py

```diff
@@ -98,14 +108,29 @@ def _upload_results_file_to_synapse(self, results_path: str) -> None:
             ),
         )
 
+    @staticmethod
+    def convert_nested_columns_to_json(
```

Comment:
I feel good about this, but what is your rationale to this being a `staticmethod`?

Reply:
I created this as a `staticmethod` so that it could be used in the Jupyter Notebooks that create the expectation suite without first instantiating the `GreatExpectationsRunner` class.

Reply:
@BWMac +1 let's keep this as is. Just for pure discussion purposes, when I use staticmethods, I often also ask: "why not just a top level helper function"? 

The biggest rationale, imo, is that the staticmethod is related to the class, and not having to instantiate the class. Are there others?

Reply:
OK, I will keep that in mind in the future. No, this is the only one like this.